nvflare/apis/fl_constant.py

```diff
@@ -412,7 +412,7 @@ class FLMetaKey:
     CURRENT_ROUND = "current_round"
     TOTAL_ROUNDS = "total_rounds"
     JOB_ID = "job_id"
-    SITE_NAME = "site_name"
+    IDENTITY = "identity"
```

Comment:
There are several places we defined this "SITE_NAME". And since these "key names" have been published in the APIs, and there's quite possibilities that customer's application are using them directly. Even if try our best to change those in our codes, we may still potentially break customer's applications.

I don't see too much values of making this name change. Even if we make a change, we may think just by adding the "IDENTITY" key, while still keeping the "SITE_NAME" to make them referencing the same.

Reply:
@yhwen thanks for the review!

Yes, we have other places that use the word SITE_NAME, I did not change those.

This whole FLMetaKey class is added by me in 2.4 just for these FLModel related operations.

So no user is using this, we can safely remove this one.

But it comes to another question, which method name you think is more appropriate "get_site_name" or "get_identity"? Or they are the same? 

@holgerroth @ZiyueXu77 you can also share your thoughts

Reply:
I like `get_identity`

Reply:
yes, get_identity sounds better